accepted/2025/mono.md

```diff
@@ -0,0 +1,108 @@
+1. It is unlikely that Mono will be changing any implementation details that would break applications.
+1. Any platform changes in Linux or Mac would handled by the Mono runtime, which allows existing applications to continue to work.
+
+MicroBuild, an internal-only tool for build infrastructure in the Developer Division uses `NuGet.exe` to acquire plug-ins.
+On Windows, it downloads the latest version of `NuGet.exe` from https://dist.nuget.org/win-x86-commandline/latest/nuget.exe.
+On Linux, it downloads the latest version of `NuGet.exe` via `apt -y install nuget`.
+There is a chance we could break Mono compatibilty, leading to widespread infrastructure issues.
+Since we won't be removing any Mono code from NuGet, it is unlikely that this would happen.
+Also, MicroBuild could pin to an older version of `NuGet.exe` if needed.
+
+Another potential risk is that not all functionality of `NuGet.exe` is available in the .NET SDK.
+The [feature availability](https://learn.microsoft.com/nuget/install-nuget-client-tools?tabs=macos#feature-availability) table indicates which features users might need before they move to the .NET SDK.
+Since we won't be intentionally breaking Mono compatibility, they would still be able to run `NuGet.exe` on if needed.
+
+## Rationale and alternatives
+<!-- Why is this the best design compared to other designs? -->
+<!-- What other designs have been considered and why weren't they chosen? -->
+<!-- What is the impact of not doing this? -->
+
+We could potentially install the Mono runtime on a Windows hosted agent and run Mono tests there which would reduce complexity a little.
+We would still want to update all documentation to indicate that its not supported.
+These tests fail occasionally which also causes friction.
+
+## Prior Art
+<!-- What prior art, both good and bad are related to this proposal? -->
+<!-- Do other features exist in other ecosystems and what experience have their community had? -->
+<!-- What lessons from other communities can we learn from? -->
+<!-- Are there any resources that are relevant to this proposal? -->
+
+We have already updated our support policy stating that Mono is not supported https://learn.microsoft.com/en-us/nuget/resources/nuget-faq#does-nuget-support-mono-.
+
+
+## Unresolved Questions
+
+<!-- What parts of the proposal do you expect to resolve before this gets accepted? -->
+<!-- What parts of the proposal need to be resolved before the proposal is stabilized? -->
+<!-- What related issues would you consider out of scope for this proposal but can be addressed in the future? -->
+
+## Future Possibilities
+<!-- What future possibilities can you think of that this proposal would help with? -->
+We could look at telemetry in the future to determine if users are still running `NuGet.exe` on Mono to determine if we could ever remove the code in the application and block execution in an unsupported scenario.
```

Comment:
The user agent should include the OS. Not sure if Mono is specifically specified there, but a combination of non windows + NuGet.exe probably indicates mono.